app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomNavigator.kt

```diff
@@ -156,6 +157,11 @@ class ChatRoomNavigator(internal val activity: ChatRoomActivity) {
         activity.overridePendingTransition(R.anim.open_enter, R.anim.open_exit)
     }
 
+    fun toProfileImageDialog(avatarUrl: String) {
```

Comment:
`toProfileImage` is enough - the "same" way we have declaring other functions here.